utils/funcexp/func_substring_index.cpp

```diff
@@ -61,7 +62,7 @@ std::string Func_substring_index::getStrVal(rowgroup::Row& row,
     if (isNull)
         return "";
 
-    int64_t count = fp[2]->data()->getIntVal(row, isNull);
+    size_t count = fp[2]->data()->getIntVal(row, isNull);
```

Comment:
Is this not a signed to unsigned conversion? Not sure if this would be a problem, just pointing it out.

Reply:
Upon further reflection, this is probably not a good idea. We should revert to int64_t.